blivet/devices/luks.py

```diff
@@ -66,26 +66,22 @@ def __init__(self, name, fmt=None, size=None, uuid=None,
 
     @property
     def raw_device(self):
-        return self.slave
-
-    @property
-    def slave(self):
```

Comment:
Both blivet-gui and anaconda use this property. I think we should deprecate this now and remove it later in 3.3 to give our users some time to adjust.

Reply:
Sounds reasonable. I'll adjust it.